src/main/java/com/scottlogic/competitionEngine/application/Application.java

```diff
@@ -1,4 +1,4 @@
-package com.scottlogic.competition_engine.application;
+package com.scottlogic.competitionEngine.application;
```

Comment:
no need for application package, this can just be in the competitionEngine package